a3p-integration/proposals/b:replace-electorate/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "agoricProposal": {
+    "type": "/agoric.swingset.CoreEvalProposal",
```

Comment:
IIUC, the plan is no longer to do a separate `CoreEvalProposal`.

The EC change should take effect as part of the next `SoftwareUpgradeProposal`. See

 - #10258